tests/kernel/update_kernel.pm

```diff
@@ -0,0 +1,209 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs maint update under test for kernel/kgraft to ltp work image
+# Maintainer: Ondřej Súkup osukup@suse.cz
+
+use 5.018;
+use warnings;
+use strict;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use qam;
+
+sub update_kernel {
+    my ($repo, $patch) = @_;
+
+    fully_patch_system;
+
+    zypper_call("ar $repo kernel-update");
+    zypper_call("ref");
+
+    if (!is_patch_needed($patch)) {
+        zypper_call("in -l -t patch $patch", exitcode => [0, 102, 103], log => 'zypper.log');
+    }
+    else {
+        die "Patch isn't needed";
+    }
+}
+
+sub mod_rpm_info {
+    my $module = shift;
+
+    script_run("rpm -qf /$module");
+    save_screenshot;
+    script_run("modinfo /$module");
```

Comment:
Sure. When you don't care about the return value just use `assert_script_run`. *If* the command fails to execute which is really unexpected but could happen you are not confusing reviewers. We had the situation more than once that some steps were always failing and reviewers got either confused looking for a failure in there or in case where the failure was searched for somewhere else logs could not be found because the script collection failed and "nobody cared".